mysql-standalone-server-5.6-ubuntu/README.md

```diff
@@ -0,0 +1,9 @@
+# MySQL Server 5.6 on Ubuntu VM
```

Comment:
Please remove 5.6 from the folder name since the version of mysql can be changed easily but not the folder name due to the url change needed